sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -161,14 +164,14 @@ def __init__(
         environment: Optional[Union[Environment, str]] = None,
         environment_variables: Optional[Dict] = None,
         resources: Optional[JobResourceConfiguration] = None,
+        queue_settings: Optional[QueueSettings] = None,
```

Comment:
Let also add flattened properties of `job_tier` and `priority` as we discussed.

Reply:
The changes will be part of Vijetha's PR or I will send a follow up PR.